docs/source/docker/integration/docker_trigger_from_api.rst

```diff
@@ -81,92 +54,138 @@ The state of the worker on the `Docker Workers <https://app.lightly.ai/docker/wo
 page should now indicate that the worker is in an idle state.
 
 
-Triggering a Job through the API
---------------------------------
-
-To trigger a new job you can click on the schedule run button on the dataset
-overview as shown in the screenshot below:
-
-.. image:: images/schedule-compute-run.png
-
-After clicking on the button you will see a wizard to configure the the parameters
-for the job.
-
-.. image:: images/schedule-compute-run-config.png
-
-In our example we use the following parameters.
-
-
-
-.. code-block:: javascript
-  :caption: Docker Config
-
-  {
-    enable_corruptness_check: true,
-    remove_exact_duplicates: true,
-    enable_training: false,
-    pretagging: false,
-    pretagging_debug: false,
-    method: 'coreset',
-    stopping_condition: {
-      n_samples: 0.1,
-      min_distance: -1
-    },
-    scorer: 'object-frequency',
-    scorer_config: {
-      frequency_penalty: 0.25,
-      min_score: 0.9
-    }
-  }
-
-
-
-.. code-block:: javascript
-  :caption: Lightly Config
-
-  {
-    loader: {
-      batch_size: 16,
-      shuffle: true,
-      num_workers: -1,
-      drop_last: true
-    },
-    model: {
-      name: 'resnet-18',
-      out_dim: 128,
-      num_ftrs: 32,
-      width: 1
-    },
-    trainer: {
-      gpus: 1,
-      max_epochs: 100,
-      precision: 32
-    },
-    criterion: {
-      temperature: 0.5
-    },
-    optimizer: {
-      lr: 1,
-      weight_decay: 0.00001
-    },
-    collate: {
-      input_size: 64,
-      cj_prob: 0.8,
-      cj_bright: 0.7,
-      cj_contrast: 0.7,
-      cj_sat: 0.7,
-      cj_hue: 0.2,
-      min_scale: 0.15,
-      random_gray_scale: 0.2,
-      gaussian_blur: 0.5,
-      kernel_size: 0.1,
-      vf_prob: 0,
-      hf_prob: 0.5,
-      rr_prob: 0
-    }
-  }
-
-Once the parameters are set you can schedule the run using a click on **schedule**.
+Create a Dataset and Triggering a Job
```

Comment:
sorry, didn't see this before but it should be either "Creating a Dataset and Triggering a Job" or "Create a Dataset and Trigger a Job".